simple_ldap_user/simple_ldap_user.module

```diff
@@ -480,16 +480,21 @@ function simple_ldap_user_sync_user_to_drupal($drupal_user) {
   // Load the LDAP user, force a cache reset.
   $ldap_user = SimpleLdapUser::singleton($drupal_user->name, TRUE);
 
-  // Nothing to sync.
   if(isset($drupal_user->uid) && $drupal_user->uid == 1) {
     // Check if user is an admin.
+    // Nothing to sync.
     return;
-  } elseif (!$ldap_user->exists) {	//For other users.
+  } 
+  else {
```

Comment:
Watch your Drupal style guide compliance here.  This should be `} else {` on a single line.